drizzlepac/runastrodriz.py

```diff
@@ -636,6 +638,8 @@ def run_driz(inlist, trlfile, calfiles, mode='default-pipeline', verify_alignmen
         _pyd_err = trlfile.replace('.tra', '_pydriz.stderr')
 
         try:
+            print("Running AstroDrizzle on: {}".format(infile))
+
```

Comment:
Convert this to a log message or delete.

Reply:
Addressed in 8514ba1